src/components/Contacts/ContactDetails/ContactDontationsTab/ConatctDonationsTab.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { GqlMockedProvider } from '../../../../../__tests__/util/graphqlMocking';
+import { useGetContactDonationsQuery } from './ContactDonationsTab.generated';
+
+const accountListId = 'account-list-1';
+const contactId = 'contact-id-1';
+
+describe('ContactDonationsTab', () => {
+  it('test query', async () => {
```

Comment:
What are you testing here? Seems like you're just testing generated code instead of any app code. Good to know that it works but idk that it really helps build confidence that our app is working correctly since we didn't write this code.

Reply:
@OzzieOrca I am only testing the query here because the view doesn't contain much logic in the view.  Once I work on the other view I will ad more view-specific tests in them.